src/aliceVision/CMakeLists.txt

```diff
@@ -38,7 +38,6 @@ endif()
 # MVS modules
 
 if(ALICEVISION_BUILD_MVS)
-  add_subdirectory(imageIO)
```

Comment:
We tried to keep a split between IO and data structures as almost all modules have dependencies to data structures.
So if we don't split it, all modules will have dependencies to IO.

But maybe this commit is a needed intermediate solution before a new change? If not, we do not want to merge mvsData and imageIO.

Reply:
We have a several files related to image IO:
- image/io.hpp   `// SfM: functions to read/writes images`
- imageIO/image.hpp  `// MVS: functions to read/write images, relies on mvsData for data structure types (rgb, Color and now Image)`
- mvsData/imageIO.hpp (image.hpp on develop) `// MVS: containing EImageFileType enum`

The idea was to start simplifying things. We will probably want to merge image types and IO at some point, but it already seems weird to have, for the MVS part, this split between `imageIO/image.hpp` and `mvsData/imageIO.hpp`. And everything that uses mvsData also uses imageIO, so the merge does not seem that problematic IMHO. But if we want to keep this `imageIO` module as the one that will stay, maybe it could be the other way around: move what's inside mvsData/imageIO.hpp into the imageIO module.